docs/source/en/model_doc/distilbert.mdx

```diff
@@ -19,13 +19,16 @@ specific language governing permissions and limitations under the License.
 <a href="https://huggingface.co/spaces/docs-demos/distilbert-base-uncased">
 <img alt="Spaces" src="https://img.shields.io/badge/%F0%9F%A4%97%20Hugging%20Face-Spaces-blue">
 </a>
+<a href="https://huggingface.co/papers/1910.01108">
+<img alt="Paper page" src="https://img.shields.io/badge/Paper%20page-1910.01108-green">
+</a>
 </div>
 
 ## Overview
 
 The DistilBERT model was proposed in the blog post [Smaller, faster, cheaper, lighter: Introducing DistilBERT, a
 distilled version of BERT](https://medium.com/huggingface/distilbert-8cf3380435b5), and the paper [DistilBERT, a
-distilled version of BERT: smaller, faster, cheaper and lighter](https://arxiv.org/abs/1910.01108). DistilBERT is a
+distilled version of BERT: smaller, faster, cheaper and lighter](https://arxiv.org/papers/1910.01108). DistilBERT is a
```

Comment:
Did you mean to link it here?

```suggestion
distilled version of BERT: smaller, faster, cheaper and lighter](https://huggingface.co/papers/1910.01108). DistilBERT is a
```

Reply:
Agree that this link should either be the HF link, or, alternatively, [a link to the paper pdf](https://arxiv.org/pdf/1907.11692)